### DISCUSSION TITLE
Can you please make  [list2table] hidden in reading mode?

### DISCUSSION THREAD

**Discussion by hyperphysin** [any-block/any-block#34] [General]
<img width="113" alt="Screenshot 2024-01-04 at 9 15 11 PM" src="https://github.com/LincZero/obsidian-any-block/assets/43456637/1742daec-4104-499d-80c1-6157a3d3429e">
<img width="218" alt="Screenshot 2024-01-04 at 9 15 21 PM" src="https://github.com/LincZero/obsidian-any-block/assets/43456637/024d9eba-66c7-4b02-89fb-4620723284ff">

In live preview mode, you see a clean table. But in reading mode, you see [list2table]. Can you please make it hidden?

**Comment by LincZero** [comment]
I fixed it in V3.0.5
